pkg/operator/admission/init.go

```diff
@@ -0,0 +1,175 @@
+package admission
+
+import (
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"os"
+)
+
+const (
+	containerName      = "rook-ceph-operator"
+	secretName         = "rook-admission-controller-secret"
+	acServiceName      = "rook-admission-controller-service"
```

Comment:
You don't need the resource type on the name. 
```suggestion
	admissionServiceName      = "rook-admission-controller"
```